### DISCUSSION TITLE
买了蓝灯专业版不能用啊

### DISCUSSION THREAD

**Discussion by JoyceLMeng** [getlantern/lantern-forum-cn#876] [Q&A]
买了两年的专业版，激活了但是用不了，授权使用专业版收不到验证码。这是怎么了啊

**Comment by JoyceLMeng** [comment]
@Mike777Khoo 可以解决一下吗

**Comment by JoyceLMeng** [comment]
代理商已退款。请问现在购买专业版有微信或支付宝渠道吗

**Reply by MaxMKMax** [reply]
@JoyceLMeng 您好. 有的, 但需要弄个专属支付链接给您. 请您提供我您的账号邮箱. 我帮您问看!

**Comment by MaxMKMax** [comment]
@JoyceLMeng 谢谢您选择我们. 感谢您支持蓝灯!
